src/OnlineSales/Controllers/ImagesController.cs

```diff
@@ -1,109 +1,109 @@
-            {
-                var uploadedImage = scopeAndFileExists!.FirstOrDefault();
-                uploadedImage!.Data = imageInBytes;
-                uploadedImage!.Size = incomingFileSize;
-
-                pgDbContext.Images!.Update(uploadedImage);
-            }
-            else
-            {
-                Image uploadedImage = new ()
-                {
-                    Name = incomingFileName,
-                    Size = incomingFileSize,
-                    Data = imageInBytes,
-                    MimeType = incomingFileMimeType!,
-                    ScopeUid = imageCreateDto.ScopeUid.Trim(),
-                    Extension = incomingFileExtension,
-                };
-
-                await pgDbContext.Images!.AddAsync(uploadedImage);
-            }
-
-            await pgDbContext.SaveChangesAsync();
-
-            Log.Information("Request scheme {0}", this.HttpContext.Request.Scheme);
-            Log.Information("Request host {0}", this.HttpContext.Request.Host.Value);
-
-            var fileData = new Dictionary<string, string>()
-            {
-                { "location", $"{Path.Combine(this.HttpContext.Request.Path, imageCreateDto.ScopeUid, incomingFileName).Replace("\\", "/")}" },
-            };
-            return CreatedAtAction(nameof(Get), new { scopeUid = imageCreateDto.ScopeUid, fileName = incomingFileName }, fileData);
-        }
-
-        [HttpGet]
-        [AllowAnonymous]
-        [Route("{scopeUid}/{fileName}")]
-        [ResponseCache(CacheProfileName = "ImageResponse")]        
-        [ProducesResponseType(StatusCodes.Status200OK)]
-        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status422UnprocessableEntity)]
-        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
-        public async Task<ActionResult> Get([Required] string scopeUid, [Required] string fileName)
-        {
-            var uploadedImageData = await (from upi in pgDbContext!.Images! where upi.ScopeUid == scopeUid && upi.Name == fileName select upi).FirstOrDefaultAsync();
-
-            if (uploadedImageData == null)
-            {
-                throw new EntityNotFoundException(nameof(Image), $"{scopeUid}/{fileName}");
-            }
-
-            return File(uploadedImageData!.Data, uploadedImageData.MimeType, fileName);
-        }
-    }
+﻿// <copyright file="ImagesController.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+
+using System.ComponentModel.DataAnnotations;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.StaticFiles;
+using Microsoft.EntityFrameworkCore;
+using OnlineSales.Data;
+using OnlineSales.DTOs;
+using OnlineSales.Entities;
+
+namespace OnlineSales.Controllers
+{
+    [Authorize]
+    [Route("api/[controller]")]
+    public class ImagesController : ControllerBase
```

Comment:
@Exmirai this needs to be changed to MediaController. If you would like /api/images to continue working to keep backward compatibility - please add temporal controller which will serve as an alias for /api/media. It could be just a simple controller class inherited from ImagesController.